helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -82,9 +82,11 @@ spec:
             mountPath: /usr/local/tomcat/shared/classes/alfresco/extension/dev-log4j.properties
             subPath: dev-log4j.properties
           {{- end }}
+          {{- if .Values.persistence.dynamicProvisioning }}
```

Comment:
I would argue we would need volume to be mounted even if not using dynamic provisioning